### ISSUE TITLE
user namespace is missing for some of the accounts (side effects is 'Internal Server Error' during che workspace creation) 

### ISSUE THREAD

**Issue by mmorhun** [openshiftio/openshift.io#3847]
When I try to start any workspace it fails right after loading GWT IDE with `Internal Server Error occurred, error time: ********` message in a popup.

I have `<username>-che` namespace. When I try to see logs or events I only can see `No Pods` message.
I can always reproduce this issue. It doesn't matter which stack I try to use (I tried a few - the same result).

**Comment by AndrienkoAleksandr** [comment]
I have the same issue.

**Comment by ibuziuk** [comment]
@AndrienkoAleksandr @mmorhun could you please provide events from `*-che` namespaces from oso ?

**Comment by ibuziuk** [comment]
for @mmorhun case the che server logs shows the following:

```
com.google.common.util.concurrent.UncheckedExecutionException: java.lang.RuntimeException: No namespace with type 'user' was found in the user tenant 	at com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2214) 	at com.google.common.cache.LocalCache.get(LocalCache.java:4053) 	at com.google.common.cache.LocalCache.getOrLoad(LocalCache.java:4057) 	at com.google.common.cache.LocalCache$LocalLoadingCache.get(LocalCache.java:4986) 	at com.redhat.che.multitenant.TenantDataProvider.getUserCheTenantData(TenantDataProvider.java:85)
--
```
@AndrienkoAleksandr could you please provide your osio username ?

**Comment by AndrienkoAleksandr** [comment]
@ibuziuk I'm using oandriie to login.

**Comment by ibuziuk** [comment]
@AndrienkoAleksandr do you have all the namespaces when you login to oso ? it appeared to be that @mmorhun is missing user namespace (`mmorhun` namespace does not exist) :

![image](https://user-images.githubusercontent.com/1461122/41843320-881310a2-786d-11e8-931f-2c6171d82a67.png)

**Comment by ibuziuk** [comment]
@aslakknutsen any ideas why user namespace could me missing ?

**Comment by AndrienkoAleksandr** [comment]
![screenshot-console starter-us-east-2 openshift com-2018 06 25-12-53-13](https://user-images.githubusercontent.com/6873095/41843673-cd3de270-7876-11e8-939d-f51a7b4322d2.png)

**Comment by ibuziuk** [comment]
@AndrienkoAleksandr O_O you're also missing user namespace `oandriie` but what is super weird is the `TestProject` namepace.

**Comment by ibuziuk** [comment]
@AndrienkoAleksandr I would really appreciate if you could try to delete `TestProject` (if you would be able to do it, this would mean that you have created the project yourself, otherwise it would mean some subtle bug in init-tenant)

**Comment by AndrienkoAleksandr** [comment]
@ibuziuk I deleted project. Look video, please:
![screencast-console starter-us-east-2 openshift com-2018 06 25-15-05-24](https://user-images.githubusercontent.com/6873095/41849703-86e38974-788a-11e8-924c-9be83ee01b78.gif)

**Comment by ibuziuk** [comment]
The issue with missing / extra namespaces is tracked [1] by the SD team. 
@mmorhun @AndrienkoAleksandr the issue for your accounts have been fixed. could you please re-login (in order to trigger `init-tenant`) and verify that the problem is fixed ?

[1] https://gitlab.cee.redhat.com/dtsd/housekeeping/issues/1852

**Comment by ibuziuk** [comment]
The issue with extra namespaces is supposed to be fixed as part of  https://github.com/openshiftio/openshift.io/issues/1446

**Comment by mmclanerh** [comment]
@mmorhun are you able to go to profile -> edit profile -> update tenant? that should create missing objects. if that doesn't work go to the same spot and try "reset environment". 

i am not seeing namespaces for mmorhun* via tier2 or via service account -- so, if they were manually created, this would be in a space that i cannot see into.

**Comment by mmorhun** [comment]
@mmclanerh yes updating of tenant helped. Thank you.

**Comment by AndrienkoAleksandr** [comment]
@mmclanerh updating of tenant helped me too. Thank you!

**Comment by ibuziuk** [comment]
@mmorhun @AndrienkoAleksandr thanks for reporting! Closing this one since the issue with extra namespaces is supposed to be fully fixed as part of #1446
